src/kOS/Suffixed/SingleResourceValue.cs

```diff
@@ -19,11 +19,9 @@ private void InitializeSuffixes()
             AddSuffix("NAME", new Suffix<string>(() => partResource.resourceName));
             AddSuffix("AMOUNT", new Suffix<double>(() => partResource.amount));
             AddSuffix("DENSITY", new Suffix<double>(() => partResource.info.density));
-            AddSuffix("UNITCOST", new Suffix<double>(() => partResource.info.unitCost));
             AddSuffix("CAPACITY", new Suffix<double>(() => partResource.maxAmount));
-            AddSuffix("TWEAKABLE", new Suffix<bool>(() => partResource.isTweakable));
-            AddSuffix("FLOWMODE", new Suffix<string>(() => partResource.flowMode.ToString()));
-            AddSuffix("FLOWSTATE", new SetSuffix<bool>(() => partResource.flowState, value =>
+            AddSuffix("TOGGLEABLE", new Suffix<bool>(() => partResource.isTweakable));
+            AddSuffix("ENABLED", new SetSuffix<bool>(() => partResource.flowState, value =>
```

Comment:
after looking at this again i realized that i unitcost was an oddball because we never show any cost information in kOS

flowstate flowmode and tweakable were flow-through properties of the resource class and on a second look i dont like the way they would be presented.